app/scripts/controllers/deploymentConfig.js

```diff
@@ -140,7 +140,7 @@ angular.module('openshiftConsole')
               }
               $scope.deploymentConfig = deploymentConfig;
 
-              if ($scope.forms.dcEnvVars.$pristine) { 
+              if (!$scope.forms.dcEnvVars || $scope.forms.dcEnvVars.$pristine) { 
```

Comment:
i saw a JS error on this so went ahead and fixed it too, i think when the controller first runs the form doesnt exist yet because the view hasnt been rendered